packages/base/src/keybindings.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "command": "jupytercad:removeObject",
+    "keys": ["Accel X"],
```

Comment:
I feel like "Accel X" should be the keybinding for "cut", though we don't have any copy/paste logic available so I'm fine keeping this keybinding for now if you like 👍🏽 

Though should we also add "Delete" there too in the list?

Reply:
Thanks, that makes much sense as xref:[QuantStack/jupytergis#68](https://github.com/QuantStack/jupytergis/pull/68/files#diff-cb12ddc72814b382343561b2b38b359975e1ea74b168abd4c0db0d4e6d9d0a2bR94-R99) also uses the `[Delete]` key for the same.

Also it sounds nice to have a feature request open for cut/copy/paste logic. I can open one if it's fine.

Reply:
> I can open one if it's fine

Sounds good!